tests/assets/test_solution_shared_lib/modules/sample_module/Dockerfile.amd64

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.1-sdk AS build-env
+FROM mcr.microsoft.com/dotnet/sdk:2.1 AS build-env
```

Comment:
I think the newest version of dotnet code sdk is up to 3.1 now, do we want to take this opportunity to upgrade?
example: FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build-env

Reply:
I prefer to tackle the upgrade separately